### DISCUSSION TITLE
raiseload for relationship of relationship

### DISCUSSION THREAD

**Discussion by aathan** [sqlalchemy/sqlalchemy#8121] [Usage Questions]
```
query(A).options(raiseload(A.b.c))
```

although c is most certainly a relationship that exists in the class of the object at the far side of the b relationship on A gives:

```
...
AttributeError: 'Comparator' object has no attribute 'c'
...
The above exception was the direct cause of the following exception:
...
AttributeError: Neither 'InstrumentedAttribute' object nor 'Comparator' object associated with A.b has an attribute 'c'
```

Generally speaking, how might I load the immediate relationships of a queried object but not the relationships of those objects?

Many thanks in advance.

**Comment by zzzeek** [comment]
you would do this like this:

```py
defaultload(A.b).raiseload(B.c)
```

[docs](https://docs.sqlalchemy.org/en/14/orm/loading_relationships.html#specifying-sub-options-with-load-options)

**Reply by zzzeek** [reply]
I can see that defaultload should be mentioned sooner outside of that section
